src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/CsvFormatResponseIT.java

```diff
@@ -136,6 +139,23 @@ public void arraysAreQuotedInFlatMode() throws IOException {
         setFlatOption(false);
     }
 
+    @Test
+    public void doubleQuotesAreEscapedWithDoubleQuotes() throws IOException {
+
+        final String query = "SELECT * FROM " + TEST_INDEX_NESTED_WITH_QUOTES;
+
+        final CSVResult csvResult = executeCsvRequest(query, false);
+        final List<String> rows = csvResult.getLines();
+        Assert.assertThat(rows.size(), equalTo(2));
+
+        final String expectedValue1 = "\"[{dayOfWeek=6, author=z\"\"z, info=zz}]\"";
+        final String expectedValue2 = "\"[{dayOfWeek=3, author=this \"\"value\"\" contains quotes, info=rr}]\"";
+
+        for (String row : rows) {
```

Comment:
instead of loop use
assertThat(rows, containsInAnyOrder(expectedValue1, expectedValue2));

(consider inlining)

Reply:
that will always fail in this case, because I am doing substring match, not the whole string.